### ISSUE TITLE
Stuck after surface rescue (and other commands)

### ISSUE THREAD

**Issue by OKAllu** [olandra/CBSafe-Simulation#17]
After completing surface rescue
- Swimmer "gets up" and walks back to shore on water (replicated multiple times)
- Everyone went back to default state (standing next to firetruck) and left the raft in the water (other time the guy on the raft stayed in the water with the raft, this happened only once)
- Everyone got stuck, not taking anymore commands (replicated multiple times, same happened with all commands)

**Comment by OKAllu** [comment]
More tests regarding this:
- Appears to be happening only after surface rescue (no problems with other commands anymore for some reason)
- Only the guy on raft gets stuck (other two accept new commands and can complete them
- Rare case: guy on raft leaves the raft in water and goes to idle state with the rest and accepts new commands like normal (raft stays in water)

**Comment by olandra** [comment]
Fixed in changeset 683. If further problems arise, please open a new issue.
